src/coll/options.rs

```diff
@@ -477,7 +489,8 @@ pub struct AggregateOptions {
     #[builder(default)]
     #[serde(
         serialize_with = "serialize_duration_as_int_millis",
-        rename = "maxTimeMS"
+        rename(serialize = "maxTimeMS"),
```

Comment:
I think this needs to be deserialized from `maxTimeMS` also

Reply:
I was generally avoiding assuming what the casing should be for these fields since they're not actually present in any of the crud spec tests, but from grepping the spec repo it seems like this one is consistently maxTimeMS elsewhere -- I'll update.